pymc4/gp/util.py

```diff
@@ -0,0 +1,75 @@
+from typing import Union
+
+import tensorflow as tf
+import numpy as np
+
+
+ArrayLike = Union[np.ndarray, tf.python.framework.ops.EagerTensor]
+TfTensor = tf.python.framework.ops.EagerTensor
+
+
+def stabilize(K, shift=1e-6):
+    r"""Add a diagonal shift to a covarience matrix"""
+    return tf.linalg.set_diag(K, tf.linalg.diag_part(K) + shift)
+
+
+def plot_gp_dist(
```

Comment:
We might want plotting functions in their own submodule, rather than pooled with other utilities. There are potentially quite a few plots that could eventually be added.